shell/common/switches.h

```diff
@@ -266,6 +266,11 @@ DEF_SWITCH(EnableVulkanValidation,
            "Enable loading Vulkan validation layers. The layers must be "
            "available to the application and loadable. On non-Vulkan backends, "
            "this flag does nothing.")
+DEF_SWITCH(ImpellerUseGL,
+           "impeller-use-gl",
```

Comment:
`--impeller-force-gl` makes it a teeny bit clearer IMO.

Reply:
Done